examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        t_test = model.train(data, sampler, num_epochs=10, batch_size=batch_size, lr=lr, val_epoch=6)
</pre><p>would it be possible to parametrize the epoch values? maybe that's one of the reasons why the tests are taking so long. If we parametrize this and control it with papermill, we will reduce the time. FYI @anargyri @aeroabir</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I have modified to have <code>num_epochs=num_epochs</code>so that epoch parameter is passed properly. That said, even for 10 epochs it does not take more than 23-24 minutes. </p>